test/types/range/elliot/anonymousRangeIter.skipif

```diff
@@ -10,3 +10,6 @@ COMPOPTS <= --no-inline
 
 # test greps generated c code, will fail for llvm
 COMPOPTS <= --llvm
+
+# --no-denormalize creates slightly differnet temporary names
```

Comment:
No they are not. I can take a look at changing the test to cover both cases, but it is not obvious to me how I can do that.